hnsecure/src/test/java/ca/bc/gov/hlth/hnsecure/parsing/FhirPayloadExtractorTest.java

```diff
@@ -1,4 +1,47 @@
 package ca.bc.gov.hlth.hnsecure.parsing;
 
-public class FhirPayloadExtractorTest {
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThrows;
+
+import java.io.UnsupportedEncodingException;
+
+import org.junit.Test;
+
+import ca.bc.gov.hlth.hnsecure.exception.CustomHNSException;
+import ca.bc.gov.hlth.hnsecure.message.ErrorMessage;
+import ca.bc.gov.hlth.hnsecure.test.TestPropertiesLoader;
+import net.minidev.json.parser.ParseException;
+
+public class FhirPayloadExtractorTest extends TestPropertiesLoader {
+	
+	private static final String fhirJsonMsg = "{\"content\":[{\"attachment\":{\"data\":\"MDAwMDAzNTJNU0h8Xn5cJnxITldFQnxWSUhBfFJBSUdULVBSU04tRE1HUnxCQzAwMDAxMDEzfDIwMTcwMTI1MTIyMTI1fHRyYWluOTZ8UjAzfDIwMTcwMTI1MTIyMTI1fER8Mi40fHwKWkhEfDIwMTcwMTI1MTIyMTI1fF5eMDAwMDAwMTB8SE5BSUFETUlOSVNUUkFUSU9OfHx8fDIuNApQSUR8fDEyMzQ1Njc4OTBeXl5CQ15QSA==\",\"contentType\":\"x-application\\/hl7-v2+er7\"}}],\"resourceType\":\"DocumentReference\",\"status\":\"current\"}";
```

Comment:
Low priority but is it better to pull these from the samples class?
It would be good to just run these all into actual files at some point.

Reply:
Agreed. I will either add these messages or attempt to re-use existing ones.